contributors/devel/sig-scalability/kubemark-setup-guide.md

```diff
@@ -0,0 +1,60 @@
+## Introduction
+This document serves to understand how to set up kubemark cluster, both inside and outside gce.
+
+## Precondition
+You need two k8s clusters(cluster one and cluster two) to set up one kubemark cluster. 
+
+The functions of the two clusters are as follows:
+
+- cluster one: its masters will be the kubemark cluster's masters.
+- cluster two: it's used to create hollow nodes for the kubemark cluster.
+
+## Steps:
+1. Build kubemark image
+
+- i. pull kubernetes code
+
+```
+cd $GOPATH/src/k8s.io/
+git clone git@github.com:kubernetes/kubernetes.git
+```
+
+- ii. build kubemark binary 
+
+```
+./hack/build-go.sh cmd/kubemark/
+cp $GOPATH/src/k8s.io/kubernetes/_output/bin/kubemark $GOPATH/src/k8s.io/kubernetes/cluster/images/kubemark/
+```
+
+- iii. build kubemark image
+
+```
+cd $GOPATH/src/k8s.io/kubernetes/cluster/images/kubemark/
+make build
+```
+
+Then you can get the image: staging-k8s.gcr.io/kubemark:latest
```

Comment:
These steps won't work by itself to use new kubemark image. They'll only locally build the docker image on your computer. When the kubemark hollow-node pods are actually cerated in your cluster, they'll pull from dockerhub (not your local computer).

So:

- either provide documentation on how to build, push and use kubemark image with custom image repository
- or get rid of this "Build kubemark image" section and just use whatever is the latest image available in the public repo `staging-k8s.gcr.io/kubemark`

Reply:
Done